### DISCUSSION TITLE
mutateFormDataBeforeSave on RelationManager

### DISCUSSION THREAD

**Discussion by loranger** [filamentphp/filament#1651] [Help] [answered]
Hi,

Does someone know if there's a way to call the `mutateFormDataBeforeSave` method from a RelationManager ?
Am I supposed to create a kind of EditRelationManager Page ?

Thanks ;)

**Comment by danharrin** [comment, marked as answer]
You can put the method straight on your relation manager! No need for extra classes.

**Reply by loranger** [reply]
So simple, I should have try !

Thanks 👍🏻
